### ISSUE TITLE
[cvpaper] CVPR2019 #104 Interpretable and Fine-Grained Visual Explanations for Convolutional Neural Networks

### ISSUE THREAD

**Issue by kazuwjnlab** [kazuwjnlab/cvpaper#101]
## \#104 [Interpretable and Fine-Grained Visual Explanations for Convolutional Neural Networks](http://openaccess.thecvf.com/content_CVPR_2019/papers/Wagner_Interpretable_and_Fine-Grained_Visual_Explanations_for_Convolutional_Neural_Networks_CVPR_2019_paper.pdf)
Jorg Wagner, Jan Mathias Kohler, Tobias Gindele, Leon Hetzel, Jakob Thaddaus Wiedemer, Sven Behnke
### どんな論文か？
特定クラスに推論された入力画像に対し，その推論根拠となるような画像箇所を強調するような最適化ベースの手法を提案した．
### 新規性
本手法は最適化中の勾配に対してフィルタを施すことでadversarial evidenceを防御する新たなテクニックに基づいており，この防御手法はハイパパラメータに依存せず，精細に画像のエッジやカラーといった特徴を捉えることができる．
### 結果
定性評価としてBBMP, Gradient, Guided Backprop,
Contrastive Excitation Backprop , Grad-CAM , Occlusion, FGVis (提案手法)を比較した．BBMP，GradCAM，Occlusionは対象物体から背景まで注目が当たってしまっているのに対して，提案手法ではピクセルレベルで注目を当てることができている．
### その他（なぜ通ったか？等）
ネットワークの構造によりかかる注目が変わる結果が得られている．
例えばResNetであればグリッド上に注目が得られるなど．
![thumb](https://drive.google.com/uc?export=view&id=1pM2-PGBFL-ctLM6dNSFOEoL9WYDEx2kK)
visual explanation,
